packages/data-context/src/data/ProjectConfigIpc.ts

```diff
@@ -17,7 +17,7 @@ const debug = debugLib(`cypress:lifecycle:ProjectConfigIpc`)
 const CHILD_PROCESS_FILE_PATH = require.resolve('@packages/server/lib/plugins/child/require_async_child')
```

Comment:
Lets fix this one as well.

Reply:
I wonder if this is actually a problem? Eg - I added regression tests, none of them fail. Kind of hesitant to add this helper unless it's actually necessary.

We use it like this:

```js
    const proc = fork(CHILD_PROCESS_FILE_PATH, configProcessArgs, childOptions)
```

I think `fork` must be smart enough to escape the path on its own.

Reply:
Edit: I see https://docs.cypress.io/guides/references/advanced-installation#Binary-cache

- Windows: /AppData/Local/Cypress/Cache

Since Cypress is always in a directory w/o a space, this has never been a problem. If it's never been a problem, I am kind of hesitant to add this change right now before shipping (without another round of manual testing and a way to reproduce the error it's meant to be fixing.